### ISSUE TITLE
Node js Application Fails on Launch Debugger

### ISSUE THREAD

**Issue by khinze** [Microsoft/vscode#39166]
- VSCode Version: 1.18.1
- OS Version: Windows 8
- Node: v9.0.0

Steps to Reproduce:

Hit f5 to launch your node js application.

Launch Configurations: 
{

            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "${workspaceRoot}\\main.js"
        },

On launch I get the following error: 
util.js:976
    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'superCtor', 'function');
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "superCtor" argument must be of type function

![error](https://user-images.githubusercontent.com/1935991/33253892-95d012a4-d34e-11e7-8d4a-5cc95e574f22.PNG)

This problem started occurring when I updated node. Is there any way this problem can be solved?

**Comment by weinand** [comment]
And your program does not crash with node.js v9.0 when running outside of VS Code?

(The exception looks like a problem/incompatibility in your program and not a problem with VS Code).

**Comment by khinze** [comment]
Hi Weinand,

Hope you well. 

I managed to resolve the issue. It did not have to do with VS Code as you suspected.

I ended up proceeding with the following:
* Updating VS Code. 
* Cleaning all my node modules 
* Reinstalling node

I did not actually suspect at the beginning that it might have been an problem/incompatibility, but I see now that from the error message it is directly referring to the spdy library which should've made me realize this initially. After digging a bit I found zetta js is dependent on this library and this is how it made its way into the solution. I am having different battle now with with spdy and zetta library. 

Thank you very much for your help, much appreciated.
